### PR TITLE
Upgrade to latest playdoh-lib

### PR DESCRIPTION
**Pull request by johngian**
* Upgrade Django to 1.6.10
* Upgrade django-compressor to v.1.4
* Remove django-jingo-offline-compressor dependency (not supported by latest django-compressor)
* Use django-compressor for offline compression

### CODE REVIEW

**Comment by akatsoulas** [comment]
I believe all the changes for the update to Django version 1.6.10 should be in one commit in order to easily revert if needed.

**Comment by akatsoulas** [comment]
Update the name of this PR to something more descriptive like 'Update to latest playdoh-lib and to Django version 1.6.10' or similar

**Comment by johngian** [comment]
I think that's better to handle dependency changes in different commits than the codebase changes.

**Comment by akatsoulas** [comment]
Although it's easier to manage PRs this way, it's not easy to revert if something goes wrong. I would favor multiple commits if there were multiple updates of multiple packages, independent of each other. Regarding the changes in the codebase, the only reason they exist is because they are needed for the update to version 1.6.10. If this not the case, then please move them to another PR. Although I don't agree for the reasons explained, it's up to you whether you are going to proceed with multiple commits or not.

File: mozillians/phonebook/forms.py [hunk 1]
```diff
@@ -157,12 +157,12 @@ class ProfileForm(happyforms.ModelForm):
     lat = forms.FloatField(widget=forms.HiddenInput)
     lng = forms.FloatField(widget=forms.HiddenInput)
     savecountry = forms.BooleanField(
-        label=_lazy('Required'),
+        label=_lazy(u'Required'),
```
Comment (akatsoulas) [inline mozillians/phonebook/forms.py]: Do you need this for the update?

Comment (johngian) [inline mozillians/phonebook/forms.py]: Yes

File: mozillians/phonebook/validators.py [hunk 2]
```diff
@@ -71,8 +71,13 @@ def validate_username_not_url(username):
 
 def validate_email(value):
     """Validate that a username is email like."""
-    if not email_re.match(value):
+    _validate_email = EmailValidator()
```
Comment (akatsoulas) [inline mozillians/phonebook/validators.py]: Why not use the `validate_email` instance?

File: mozillians/phonebook/forms.py (hunk 1 above)
Comment (johngian) [inline mozillians/phonebook/forms.py]: Yes, without this ``_lazy`` raises ``TypeError("Lazy object returned unexpected type.")``.

**Comment by akatsoulas** [comment]
I tested this locally and everything seems to be working fine. r+wc

File: mozillians/phonebook/validators.py (hunk 2 above)
Comment (johngian) [inline mozillians/phonebook/validators.py]: To keep it consistent with the rest of the validators. We could use validate_email and import with a different name, but it does pretty much the same thing I am doing here.

**Comment by johngian** [comment]
Squashed commits to a single one.

**Pull request merged**